orc8r/cloud/go/services/bootstrapper/servicers/registration/cloud_registration_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package registration_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"magma/orc8r/cloud/go/services/bootstrapper"
+	"magma/orc8r/cloud/go/services/bootstrapper/servicers/registration"
+	"magma/orc8r/cloud/go/test_utils"
+	"magma/orc8r/lib/go/protos"
+
+	"github.com/stretchr/testify/assert"
+)
```

Comment:
Let's also test that IsExpired works correctly. Easiest way to do this is probably to use clock